sdk/lib/atl/atlwin.h

```diff
@@ -575,6 +577,18 @@ class CWindow
         return ::GetDlgItemText(m_hWnd, nID, lpStr, nMaxCount);
     }
 
+    UINT GetDlgItemText(int nID, CSimpleString& string)
+    {
+        HWND item = GetDlgItem(nID);
+        int len = ::GetWindowTextLength(item);
+        if (len > 0)
```

Comment:
I dont think we need this condition. (and in case where len=0, your string could be non-empty)